### ISSUE TITLE
File with fully qualified path name cannot be found

### ISSUE THREAD

**Issue by marcelpaulo** [iamleot/transferwee#4]
If the file has a fully qualified path name, such as */tmp/hello.txt*, it cannot be found. For instance, I created this file:

```
paulo:~/src/transferwee (master)$ ls -l /tmp/hello.txt
-rw-rw-r-- 1 paulo paulo 12 mai 31 13:11 /tmp/hello.txt
```

and when I try to upload it, I get this error:

```
paulo:~/src/transferwee (master)$ python3 transferwee.py upload -f marcelpaulo@gmail.com -t marcelpaulo@gmail.com -- /tmp/hello.txt
Traceback (most recent call last):
  File "transferwee.py", line 325, in <module>
    print(upload(args.files, args.m, args.f, args.t))
  File "transferwee.py", line 275, in upload
    _prepare_email_upload(filenames, message, sender, recipients)['id']
  File "transferwee.py", line 149, in _prepare_email_upload
    "files": [_file_name_and_size(f) for f in filenames],
  File "transferwee.py", line 149, in <listcomp>
    "files": [_file_name_and_size(f) for f in filenames],
  File "transferwee.py", line 133, in _file_name_and_size
    filesize = os.path.getsize(file)
  File "/usr/lib/python3.6/genericpath.py", line 50, in getsize
    return os.stat(filename).st_size
FileNotFoundError: [Errno 2] No such file or directory: 'hello.txt'
```

If I run `transferwee` from the same directory as the file, it works:

```
paulo:~/tmp$ python3 ~/src/transferwee/transferwee.py upload -f marcelpaulo@gmail.com -t marcelpaulo@gmail.com -- hello.txt
https://we.tl/t-cuohfc9FOP
```

**Comment by iamleot** [comment]
Thank you for reporting this problem!

Commit 3b3afe4 should fix it.

Please let me know if it's okay to close this issue or feel free to close it if that's okay!

**Comment by marcelpaulo** [comment]
Thanks a lot, @iamleot, for the speed-lightning fix! I've just tried it, working really fine!

Your script is really useful. I use quodlibet to manage my music, and very frequently I send albums to friends using WeTransfer. I whipped up a script to zip an album and I manage to call it from quodlibet using an extension. But then I had to load WeTransfer on a browser to send the zipped album. Now I'll be able to insert transferwee into the script and send an album directly from quodlibet. Thanks a lot for all your work!
